src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/RegistrationValidators.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Protocol;
+using System.Threading.Tasks;
+using NuGet.Protocol.Core.Types;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring.Validation.Test
+{
+    public class RegistrationValidators
+    {
+        public static string GetContainsString(bool contains)
+        {
+            return contains ? "contains" : "doesn't contain";
+        }
+        
+        /// <summary>
+        /// Verifies that the index.json blob for the package exists when it should.
+        /// </summary>
+        public class RegistrationIndexExistsValidator : Validator, IValidator<RegistrationEndpoint>
```

Comment:
>RegistrationEndpoint [](start = 78, length = 20)

Could the type parameter be on Validator? That is, won't these validators always have exactly on endpoint?